action/rlp_tx.go

```diff
@@ -138,6 +136,12 @@ func ExtractTypeSigPubkey(tx *types.Transaction) (iotextypes.Encoding, []byte, c
 			encoding = iotextypes.Encoding_ETHEREUM_UNPROTECTED
 			signer = types.HomesteadSigner{}
 		}
+	case types.AccessListTxType:
+		// AL txs are defined to use 0 and 1 as their recovery
+		// id, add 27 to become equivalent to unprotected Homestead signatures.
+		V = new(big.Int).Add(V, big.NewInt(27))
+		encoding = iotextypes.Encoding_ETHEREUM_ACCESSLIST
+		signer = types.NewEIP2930Signer(tx.ChainId())
```

Comment:
https://github.com/ethereum/go-ethereum/blob/a36c68f12cdc45f52184b563dab475f340e00dd7/core/types/transaction_signing.go#L298

Reply:
can you be more specific what you mean by quoting this code?